src/containers/Header/__snapshots__/Header.test.jsx.snap

```diff
@@ -1,269 +0,0 @@
-// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
You don't need a new Snap?

Reply:
I removed all snapshot tests from this component in favor of unit tests. We've kind of agreed as a group that snapshot tests aren't serving our purposes well. I actually added two more unit tests that capture things we weren't getting from the snapshots.